src/webgpu/shader/execution/memory_model/memory_model_setup.ts

```diff
@@ -277,6 +299,26 @@ export class MemoryModelTester {
       size: shuffledWorkgroupsSize,
     };
 
+    if (this.useTexture) {
+      const numTexels = testLocationsSize / bytesPerWord;
```

Comment:
nit: Should these divisions use `floor` to ensure integral results?

Reply:
This is guaranteed earlier in the flow.